main.tf

```diff
@@ -5,12 +5,59 @@ resource "aws_instance" "web" {
   tags = {
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure IMDv1 is disabled for AWS EC2 instances**
 
**Vulnerability**
AWS EC2 instances that have IMDv1 enabled are vulnerable to server side request forgery (SSRF) attacks.
 
**Remediation**
IMDv1 is vulnerable to SSRF attack. Thus, EC2 Instance should have metadata_options block with either http_endpoint attribute set to disabled or ensure http_tokens attribute is set to required. This ensures IMDv2 is being used.

source: accurics-stageone